uncrustify.cfg

```diff
@@ -55,14 +59,19 @@ sp_pp_concat				= ignore
 sp_pp_stringify				= ignore
 sp_bool						= force
 sp_after_class_colon    	= force
-sp_before_class_colon   	= remove
+sp_before_class_colon   	= force
+sp_after_constr_colon		= force
 sp_cmt_cpp_start			= add
 sp_cmt_cpp_doxygen			= true
+sp_endif_cmt				= force
 sp_angle_shift				= remove
 sp_permit_cpp11_shift		= true
 sp_before_sparen			= force
 sp_after_operator			= remove
 sp_after_operator_sym		= remove
+sp_after_for_colon			= force
+sp_before_for_colon			= force
+sp_between_new_paren		= force
```

Comment:
@evsh:  And do not even get me started on how "all over the place" the spacing of these lines is…

Reply:
Same as the very first reply.